.github/workflows/ci-review-trigger.yml

```diff
@@ -37,30 +40,30 @@ env:
 
 # The below concurrency group settings would let us cancel in progress runs that were triggered with the
 # same comment on a given PR, which could save on time consuming runs.
-# But GH does not currently support the github.event.comment.body as part of the concurrency name, this
+# But GH does not currently support the github.event.review.body as part of the concurrency name, this
 # appears to be due to the potential length of it.
 #concurrency:
-#  group: ${{ github.workflow }}-${{ github.event.issue.id }}-${{ github.event.comment.body }}
+#  group: ${{ github.workflow }}-${{ github.event.issue.id }}-${{ github.event.review.body }}
 #  cancel-in-progress: true
 
 jobs:
   validate:
```

Comment:
do you need a step here to validate whether the commit it's running on is the latest in the PR? Some labelling and other actions might give a false sense of what has been tested...

Reply:
I don't think it is necessary that the commit be the latest commit in the PR. The workflows will run again as part of the merge queue so if there are breakages they will be caught then. This comment trigger workflow is just to have the ability to trigger them without adding the PR to the merge queue.

An exception is the component features check which doesn't run on the merge queue, instead it runs nightly, but breakages to that are low urgency and can be resolved later.